infra-as-code/bicep/modules/policy/assignments/alzDefaults/alzDefaultPolicyAssignments.bicep

```diff
@@ -101,6 +101,9 @@ param parExcludedPolicyAssignments array = []
 @sys.description('Set Parameter to true to Opt-out of deployment telemetry')
 param parTelemetryOptOut bool = false
 
+@sys.description('Effect type for all policy definitions')
+param parPolicyEffect string = 'Deny'
```

Comment:
I feel this name is too vague and could be interpreted in a variety of ways. Perhaps we need to use the 'sovereignty' keyword somewhere in the variable name.

Reply:
Updated with Sovereignty.